src/main/java/frc/robot/Robot.java

```diff
@@ -24,26 +24,29 @@
 public class Robot
 {
   // SUBSYSTEMS //
+  public Driver driver;
```

Comment:
Maybe put `driver` under a separate comment called `CONTROLS` or something like that.